Automattic/newspack-newsletters
# fix(usage-reports): handle no lists from MC

### All Submissions:






### Changes proposed in this Pull Request:



A small tweak. If there's a wrong API key set, the call to `Newspack_Newsletters_Mailchimp_Usage_Reports::get_usage_report` will log a PHP Warning.

### How to test the changes in this Pull Request:

1. On `trunk`, configure MC with a wrong API key
1. Run `wp eval "Newspack_Newsletters_Mailchimp_Usage_Reports::get_usage_report();"`
2. Observe a PHP Warning logged
3. Switch to this branch, run the snippet again, observe no warning logged

### Other information:
